src/app/components/contact-modal/contact-modal.component.html

```diff
@@ -1,69 +1,40 @@
 <div [@hideShow]="loading || hideForm ? 'hidden' : 'visible'" class="prose">
-  <ng-container *ngIf="!hideForm">
-    <h1 class="font-bold text-lg text-blue-200 mb-3">Contact opnemen</h1>
-    <p>
-      Vul hier uw e-mailadres of telefoonnummer en en de boodschap die u wilt versturen.
-    </p>
-    <form
-      [formGroup]="contactForm"
-      data-test="form-register"
-      (submit)="onClickSendEmail()"
-    >
-      <littil-form-input-text
-        formControlName="contactInfo"
-        id="contactInfo"
-        label="Contactgegevens"
-        [hasError]="FormUtil.InValid(contactForm.controls['contactInfo'])"
-      >
-        <littil-form-error-message
-          *ngIf="
-            FormUtil.ContainsError(contactForm.controls['contactInfo'], 'required')
-          "
-          errorText="Contact informatie is verplicht"
-        ></littil-form-error-message>
-      </littil-form-input-text>
-      <littil-form-input-text
-        formControlName="message"
-        id="message"
-        label="Bericht"
-        [hasError]="FormUtil.InValid(contactForm.controls['message'])"
-      >
-        <littil-form-error-message
-          *ngIf="
-            FormUtil.ContainsError(contactForm.controls['message'], 'required')
-          "
-          errorText="Message is verplicht"
-        ></littil-form-error-message>
-      </littil-form-input-text>
-    </form>
+  @if(!hideForm){
+  <h1 class="font-bold text-lg text-blue-200 mb-3">Contact opnemen</h1>
```

Comment:
check even of er marges moeten worden toegevoegd aan de randen van het modal.

![image](https://github.com/user-attachments/assets/e2e983ba-e195-4ef3-8608-9a7b8a46ac9f)

Reply:
Fixed